src/utils/typography.js

```diff
@@ -43,123 +43,123 @@ const options = {
 			marginBottom: '2em'
 		},
 		// gatsby-remark-prismjs styles
-		".gatsby-highlight": {
+		'.gatsby-highlight': {
 			background: 'rgba(0,0,0,0.04)',
-			position: `relative`,
-			WebkitOverflowScrolling: `touch`,
+			position: 'relative',
+			WebkitOverflowScrolling: 'touch'
 		},
 		".gatsby-highlight pre[class*='language-']": {
```

Comment:
Should we match quote styles with L46 so that this would be `'.gatsby-highlight pre[class*="language-"]'`?

Reply:
I don't feel strongly enough about this to change it.

Reply:
:nothingtodohere: